packages/react-native-reanimated/tsconfig.json

```diff
@@ -5,5 +5,5 @@
       "react-native-reanimated": ["./src"]
     }
   },
-  "include": ["src"]
+  "include": ["src", "metro-config"]
```

Comment:
Why is it needed?

Reply:
Because of `index.d.ts` in the `metro-config` directory

<img width="578" alt="Screenshot 2024-08-13 at 11 36 31" src="https://github.com/user-attachments/assets/58c0ba7e-1df4-44a9-8216-2f83513044e8">

Reply:
I can also add it to eslintignore but it seems better to keep eslint and ts checks